src/index.ts

```diff
@@ -228,6 +254,10 @@ async function appBoot() {
 
 		createMainWindow();
 
+		// Handle CLI commands
+		listenCLICommands();
+		executeCLICommand();
```

Comment:
` isCLI` function only checks if the current instance was launched with a CLI command. `listenCLICommands` needs to be run on the main instance to set up the `second-instance` event, which is invoked when a second instance is launched. Hence, it needs to be executed independently if a CLI command is passed in the main instance. I'd like to note that this scenario doesn't apply to `wp-cli` commands as they are meant to be executed in the same instance they are triggered. However, in the future, it might be interesting to run commands on the first instance to manage sites.

Here's a diagram that explains the process:
![image](https://github.com/Automattic/studio/assets/14905380/c752a398-ca03-4ac6-9c7d-9cbde6c5590f)

`executeCLICommand` could be wrapped with `if ( isCLI() )` but the function already checks if a CLI command exists.